force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -42,7 +52,14 @@ public with sharing class FlowControlLoopsCollections {
      * @return true if the person is eligible to vote, false otherwise.
      */
     public static Boolean votingEligibility(Integer age) {
-        return null; // Replace null with the variable you used to store the result
+        Boolean canTheyVote;
+        if(age >= 18){
```

Comment:
This is another method you can simplify by just returning the boolean expression.